### DISCUSSION TITLE
Quick question, How can I upgrade the amout of storage I have in my postgresDB?

### DISCUSSION THREAD

**Discussion by masonc-dev** [vercel/community#4625] [Help] [answered]
### Summary

Quick question, How can I upgrade the amout of storage I have in my postgresDB? I look at Vercel's paid plans but I did not see anything about how to get more postgres storage.

### Steps to Reproduce

You make a postgresDB and use the space available for free!

**Comment by swarnava** [comment, marked as answer]
Hi,

You can check Storage pricing in our Vercel Pricing page: https://vercel.com/pricing. It is "pay as you go" plan currently based on the amount of storage you use.
